src/draive/instructions/volatile.py

```
@@ -37,7 +37,11 @@ async def instruction(
         **extra: Any,
     ) -> Instruction:
         if instruction := self._storage.get(name):
-            return instruction
+            if arguments:
+                return instruction.updated(arguments={**instruction.arguments, **arguments})
+
+            else:
+                return instruction
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)